### ISSUE TITLE
vue-a11y/media-has-caption errantly fails for muted

### ISSUE THREAD

**Issue by calling** [maranran/eslint-plugin-vue-a11y#30]
**Describe the bug**
`<video muted />` should not produce an error

**To Reproduce**
1. Enable `vue-a11y/media-has-caption`
2. Add `<video muted />`
3. See error

**Expected behavior**
It should not produce an error

**Desktop (please complete the following information):**
 - Node: v12.15.0

**Additional context**
Workaround is to use `<video :muted="true" />`
